### PR TITLE
Add export macro to classes in CBridge and JNI

### PR DESCRIPTION
**Pull request by DanielKamkha**
Signed-off-by: Daniel Kamkha <daniel.kamkha@here.com>

### CODE REVIEW

File: gluecodium/src/main/resources/templates/jni/utils/JniClassCacheHeader.mustache [hunk 1]
```diff
@@ -52,7 +52,7 @@ private:
 };
 
 template <class ... CppTypes>
-class CachedJavaClass : public CachedJavaClassBase
+class JNIEXPORT CachedJavaClass : public CachedJavaClassBase
```
Comment (Hsilgos) [inline gluecodium/src/main/resources/templates/jni/utils/JniClassCacheHeader.mustache:55]: class CachedJavaClassBase should be exported, this one - not.

File: gluecodium/src/main/resources/templates/jni/utils/JniExceptionThrowerHeader.mustache [hunk 2]
```diff
@@ -30,7 +30,7 @@ namespace {{.}}
 {{/internalNamespace}}
 namespace jni
 {
-    class JniExceptionThrower
+    class JNIEXPORT JniExceptionThrower
```
Comment (Hsilgos) [inline gluecodium/src/main/resources/templates/jni/utils/JniExceptionThrowerHeader.mustache:33]: This class is header only and shouldn't be exported

File: gluecodium/src/main/resources/templates/jni/utils/JniClassCacheHeader.mustache (hunk 1 above)
Comment (DanielKamkha) [reply gluecodium/src/main/resources/templates/jni/utils/JniClassCacheHeader.mustache:55]: Done

File: gluecodium/src/main/resources/templates/jni/utils/JniExceptionThrowerHeader.mustache (hunk 2 above)
Comment (DanielKamkha) [reply gluecodium/src/main/resources/templates/jni/utils/JniExceptionThrowerHeader.mustache:33]: Done

**Review by Hsilgos — APPROVED** [review]

**Pull request merged**